exposed/src/main/kotlin/org/jetbrains/exposed/sql/transactions/TransactionApi.kt

```diff
@@ -90,4 +92,6 @@ interface TransactionManager {
 
         fun isInitialized() = managers.first != NotInitializedManager
     }
-}
\ No newline at end of file
+}
+
+val Database?.transactionManager: TransactionManager get() = TransactionManager.managerFor(this) ?: throw RuntimeException("database ${this} don't have any transaction manager")
```

Comment:
This function will return default manager when calling on `null`, is it expected?